packages/plugins/RSS3/src/SNSAdaptor/modals/DetailsModal/index.tsx

```diff
@@ -26,13 +27,15 @@ export const FeedDetailsModal = forwardRef<SingletonModalRefCreator<FeedDetailsM
 
         if (!open) return null
         return (
-            <FeedDetailsDialog
-                open
-                onClose={() => dispatch?.close()}
-                {...props_}
-                type={props_?.type ?? CardType.UnknownIn}
-                feed={props_?.feed!}
-            />
+            <ScopedDomainsContainer.Provider>
+                <FeedDetailsDialog
+                    open
+                    onClose={() => dispatch?.close()}
+                    {...props_}
+                    type={props_?.type ?? CardType.UnknownIn}
+                    feed={props_?.feed!}
+                />
+            </ScopedDomainsContainer.Provider>
```

Comment:
If `FeedsPage` and `FeedDetailsDialog` are wrapped in individual components, we cannot guarantee consistent results between them!